src/EditorFeatures/CSharp/RenameTracking/CSharpRenameTrackingLanguageHeuristicsService.cs

```diff
@@ -11,7 +11,7 @@ internal sealed class CSharpRenameTrackingLanguageHeuristicsService : IRenameTra
     {
         public bool IsIdentifierValidForRenameTracking(string name)
         {
-            return name != "var" && name != "dynamic";
+            return name != "var" && name != "dynamic" && name != "_";
```

Comment:
FYI: 'unmanaged' is another one of those special names.  as is 'nameof'.  Possibly things to look into.

Reply:
:arrow_right: Filed as #34406

Reply:
Thanks!